OAK-Marketplace-Emotion-Recognition/README.md

```diff
@@ -0,0 +1,54 @@
+This contains the code for **Emotion Recognition using OpenCV AI KIT and Modelplace**. For more information - visit [OpenCV.AI](https://www.opencv.ai/).
+
+# OAK Emotion Recognition 
+
+![](images/demo.gif)
+
+## Requirements
+- Install Python 3.7 or higher and run: ```python3 -m pip install -r requirements.txt```
+
```

Comment:
- Install [Emotion Recognition OAK model](https://github.com/opencv-ai/opencv-blog/tree/main/OAK-Marketplace-Emotion-Recognition#installation-of-emotion-recognition-oak-package)